students/AlyssaHong/Lesson04/mailroom_part2.py

```diff
@@ -0,0 +1,109 @@
+"""
+Author: Alyssa Hong
+Date: 11/04/2018
+Update:
+Lesson4 Assignments > Mailroom, Part 2
+"""
+
+#!/usr/bin/env python3
+
+import os
+
+
+list_col = ['Donor Name','Total Given','Num Gifts','Average Gift']
+#list of donation : donor_name,[total_given,num_gifts,avg_gift]
+list_donation = {'Fred':[7000,4500],'Alex':[30000,30000,10000],\
+                'Henry':[5000],'Alyssa':[120000,30000,40000],\
+                'Leo':[107000,53500]}
+
+
+def send_thanks():
+    user_input = input("Do you want to see the list of donor? Please type 'list'--> ")
+    if user_input == "list":
+        show_list_donation(list_donation)
+        donor_name = input("Type the donor's name: ")
+        new_donation = int(input("Type your donations: "))
+        if donor_name not in list_donation.keys():
+            list_donation.update({donor_name: [new_donation]})
+        else:
+            list_donation[donor_name] += [new_donation]
+        #print the email
+        print('\n' + 'Dear {:s}, thank you for your ${:.2f} donation!'.format(donor_name,new_donation) + '\n')
+        #return to the original prompt
+        show_list_donation(list_donation)
+    else:
+        print("Please type 'list' or quit")
+
+    return_menu()
+
+
+def create_report():
+    show_list_donation(list_donation)
+    return_menu()
```

Comment:
you don't need any of the `return_menu` calls - you have a `while True` loop so the loop will continue after the function finishes the execution.